etp-core/etp-db/src/test/sql/migration/repeatable/r-x-01-test-data.sql

```diff
@@ -0,0 +1,592 @@
+	(6, 'Ulla', 'Specimen-Pirix', 'ulla.laapotti@ara.fi', '0451234567', false, 0, NULL, false, '040265-9985', NULL, NULL, NULL, NULL, false, NULL, '', NULL, NULL),
+	(7, 'Kirsi', 'Juutilainen', 'kirsi.unhonen@ara.fi', '0451234567', false, 0, NULL, false, '260991-999R', NULL, NULL, NULL, NULL, false, NULL, '', NULL, NULL),
+	(8, 'Ari', 'Manninen', 'ari.manninen@ara.fi', '0451234567', false, 0, NULL, false, '010101-123N', NULL, NULL, NULL, NULL, false, NULL, '', NULL, NULL),
+	(9, 'Risto', 'Jesoi', 'risto.jesoi@ara.fi', '0451234567', false, 0, NULL, false, '180883-998N', NULL, NULL, NULL, NULL, false, NULL, '', NULL, NULL),
+	(10, 'Lamitor', 'Laatija', 'laatija@lamit.fi', '0451234567', false, 0, NULL, false, '110106A998M', NULL, NULL, NULL, NULL, false, NULL, '', NULL, NULL),
+	(11, 'Riuska', 'Laatija', 'laatija@granlund.fi', '0451234567', false, 0, NULL, false, '290574-9981', NULL, NULL, NULL, NULL, false, NULL, '', NULL, NULL),
+	(12, 'Cadmatic', 'Laatija', 'laatija@cadmatic.com', '0451234567', false, 0, NULL, false, '050391-999B', NULL, NULL, NULL, NULL, false, NULL, '', NULL, NULL),
+	(13, 'Timbal', 'Laatija', 'laatija@timbal.fi', '0451234567', false, 0, NULL, false, '120997-9998', NULL, NULL, NULL, NULL, false, NULL, '', NULL, NULL),
+	(14, 'Etlas', 'Laatija', 'laatija@etlas.fi', '0451234567', false, 0, NULL, false, '271258-9988', NULL, NULL, NULL, NULL, false, NULL, '', NULL, NULL),
+	(15, 'Caverion', 'Laatija', 'laatija@caverion.fi', '0451234567', false, 0, NULL, false, '010170-960F', NULL, NULL, NULL, NULL, false, NULL, '', NULL, NULL),
+	(16, 'Laskentapalvelut', 'Laatija', 'laatija@dof.fi', '0451234567', false, 0, NULL, false, '010170-999R', NULL, NULL, NULL, NULL, false, NULL, '', NULL, NULL),
+	(17, 'Laskentaohjelmat', 'Laatija', 'laatija@example.com', '0451234567', false, 0, NULL, false, '081181-9984', NULL, NULL, NULL, NULL, false, NULL, '', NULL, NULL),
+	(19, 'Paavo', 'Pääkäyttäjä', 'paakayttaja2@solita.fi', '0501234567', false, 2, NULL, false, NULL, NULL, 'vvirkamies3', 'testivirasto.fi', NULL, false, NULL, '', NULL, NULL),
+	(20, 'Lasse', 'Laskuttaja', 'laskuttaja@solita.fi', '0501234567', false, 3, NULL, false, NULL, NULL, 'vvirkamies2', 'testausvirasto.fi', NULL, false, NULL, '', NULL, NULL),
+	(3, 'Kalevi', 'Specimen-Potex', 'kumppani@solita.fi', '0451234568', false, 0, NULL, false, '141199-999N', NULL, NULL, NULL, NULL, false, NULL, '', NULL, NULL),
+	(18, 'Päivi', 'Pääkäyttäjä', 'paakayttaja@solita.fi', '0501234567', false, 2, '2024-02-22 12:08:23.509376+00', false, NULL, 'paakayttaja@solita.fi', 'vvirkamies', 'testivirasto.fi', NULL, false, NULL, '', NULL, NULL),
+	(2, 'Liisa', 'Specimen-Potex', 'laatija@solita.fi', '0451234567', false, 0, '2024-02-22 12:09:41.497552+00', false, '010469-999W', 'laatija@solita.fi', NULL, NULL, '2024-02-22 12:09:39.603165+00', false, NULL, '', NULL, NULL) ON CONFLICT DO NOTHING;
```

Comment:
Tässä Liisa Specimen-Potexin `verifytime` on '2024-02-22 12:09:39.603165+00'.

Tulee tuo cypress testi varmaankin hajoamaan 6kk päästä
```js
export const isVerified = kayttaja =>
  Maybe.fold(
    false,
    verifytime => dfns.isAfter(dfns.addMonths(verifytime, 6), new Date()),
    kayttaja.verifytime
  );
```
 En nyt sitten tiedä tuleeko tuo olemaan suuri ongelma, mutta yksi vaihtoehto voisi olla tehdä repeatable migration, joka käy asettamassa sen tämän. Toinen varmaan olisi tuolla cypress testissä interceptata whoami ja laittaa nulliksi tai eiliseksi.

Reply:
Oisko näin hyvä?

Reply:
Kyllä ainakin mun puolesta 👍

Reply:
Tai no, nyt nuo kyllä hajoaa lokaalisti, jos ajaa kahdesti putkeen 🤔

Reply:
Voisi varmaan olla hyvä, jos tehtäisiin erikseen e2e ja dev migraatiot, niin ei tarvitsisi varoa, että vahingossa hajottaa e2e testejä, jos haluaa jotain tietynlaista dataa laittaa dev ympäristöön, mutta ehkä se ei ole tämän ticketin homma?

Reply:
> Tai no, nyt nuo kyllä hajoaa lokaalisti, jos ajaa kahdesti putkeen 🤔
Hyvä huomio, lisätty `resetDb`